test/format/CONFIG.disagg

```diff
@@ -0,0 +1,22 @@
+############################################
+#  RUN PARAMETERS: V3
+############################################
+# A configuration for disaggregated storage.
+backup=0
+disagg.page_log=palm
+disagg.enabled=1
+disagg.layered=1
+runs.source=layered
+runs.type=row-store
+import=0
+ops.alter=0
+# SLS-2872 should fix cursor->modify for SLS.
+ops.pct.modify=0
+ops.compaction=0
+ops.salvage=0
+ops.throttle=0
+ops.truncate=0
+ops.verify=0
```

Comment:
Do we have a ticket for verify working with disagg? If so, that'd be worth linking here. (I'm being lazy here, I did a quick search for "verify" in the SLS project and nothing came up. So a `TODO` comment would be fine as well.)

Reply:
Thanks!